app/app.models.js

```diff
@@ -72,10 +68,27 @@ exports.selectArticlesById = (article_id) => {
   }
 
   return db.query(queryString, queryValue).then((result) => {
-    if(result.rows.length === 0){
-      return Promise.reject({status: 404, msg: 'not found'})
+    if (result.rows.length === 0) {
+      return Promise.reject({ status: 404, msg: "not found" });
     }
-return result.rows[0];
+    return result.rows[0];
   });
 };
 
+exports.selectComment = (article_id) => {
+  queryString = `SELECT * FROM comments`;
+  queryValue = [];
+
+  if (article_id) {
+    queryString += ` WHERE article_id = ${article_id} 
+    ORDER BY created_at ASC
```

Comment:
do you want the created_at ascending by default?

Reply:
thanks ;)

Reply:
:)*